src/router/routes.js

```diff
@@ -34,7 +34,7 @@ import Certificates from '@/views/SecurityAndAccess/Certificates';
 import VirtualMedia from '@/views/Operations/VirtualMedia';
 import Memory from '@/views/ResourceManagement/Memory';
 import Power from '@/views/ResourceManagement/Power';
-import SystemParameters from '@/views/ResourceManagement/systemParameters';
+import SystemParameters from '@/views/ResourceManagement/SystemParameters';
```

Comment:
this one is not related but the code could not built because of this error. Thought of pushing rather ignoring.